### PR TITLE
Load static assets from CDN

### PR DESCRIPTION
**Pull request by dsamojlenko**
# What it do
Adds a template helper for loading static assets from the CDN.

## Environment variable
To work, the app needs to be in production mode, and the CDN_PREFIX env variable should be set. ie:

```
CDN_PREFIX=//cv19-benefits-cdn.azureedge.net
```

## Using the asset helper
To use the helper in a nunjucks template, simply pass the static asset path into the helper:

```
<link rel="stylesheet" href="{{ asset('/dist/css/styles.css') }}">
```

If we're in production mode, the helper will return the path prefixed by the CDN url, offloading the request to the CDN. Otherwise, it will just return the path and load the asset locally.

@TODO: figure out how to refresh the CDN cache on deployment #284

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -84,6 +84,21 @@ app.use(compression())
...
+ * If a CDN_PREFIX is set in env, and mode is production,
+ * the helper will return the path with the CDN prefix,
+ * otherwise it just returns the path
+ */
+app.locals.asset = (path) => {
+  const cdnprefix = process.env.CDN_PREFIX || '';
+
+  if (process.env.NODE_ENV === 'production') {
+    return cdnprefix + path;
+  }
```
Comment (CalvinRodo) [inline app.js:98]: Would we ever want to use a CDN not in production mode?

**Comment by CalvinRodo** [comment]
Is there any point in changing our all of our inline svgs to `.svg` files for use in the CDN?

File: app.js (hunk 1 above)
Comment (dsamojlenko) [reply app.js:98]: I don't think so .... but I suppose I could change this to check for the existence of the env var, so you could just add it to whatever environments you want to load assets from the cdn?

Comment (CalvinRodo) [reply app.js:98]: No your right that was a foolish question we would want to be compile our assets as we go.

**Pull request merged**